src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/ContestedDraftOrderNotApprovedController.java

```diff
@@ -105,4 +113,33 @@
 
         return ResponseEntity.ok(AboutToStartOrSubmitCallbackResponse.builder().data(caseData).build());
     }
+
+    @PostMapping(path = "/contested-application-send-refusal", consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "Sends refusal reason to paper cases")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Callback was processed successfully or in case of an error message is attached to the case",
+            response = AboutToStartOrSubmitCallbackResponse.class),
+        @ApiResponse(code = 400, message = "Bad Request"),
+        @ApiResponse(code = 500, message = "Internal Server Error")})
+    public ResponseEntity<AboutToStartOrSubmitCallbackResponse> sendRefusalReason(
+        @RequestHeader(value = AUTHORIZATION_HEADER) String authorisationToken,
+        @NotNull @RequestBody @ApiParam("CaseData") CallbackRequest callback) {
+
+        CaseDetails caseDetails = callback.getCaseDetails();
+        log.info("Received request for send refusal reason for paper cases with Case ID: {}", caseDetails.getId());
+        validateCaseData(callback);
+
+        if (CommonFunction.isPaperApplication(caseDetails.getData())) {
```

Comment:
Could this be isContestedPaperCase()? Not sure if this would make a difference as the field for "isPaperCase" may be the same across consented and contested, and this callbakc is only ever called from contested